### PR TITLE
Introduce connection succeed modal

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**
Screenshot 2019-11-14 at 13 43 51

### CODE REVIEW

File: universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx [hunk 1]
```diff
@@ -3,24 +3,28 @@ import {useHistory} from 'react-router';
...
+        name={name!}
+        sdk={sdk}
+        walletService={walletService}
+        onCancel={() => setConnectModal('selector')}
+        onSuccess={() => setConnectModal('connectionSucceed')}
+        className="jarvis-styles"
+      />;
+    case 'connectionSucceed':
+      return <ConnectionSuccess hideModal={() => {history.push('/');}}/>;
+    default:
```
Comment (Marik-D) [inline universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx]:
```suggestion
    case 'selector':
```

File: universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx [hunk 2]
```diff
@@ -3,24 +3,28 @@ import {useHistory} from 'react-router';
...
+      return <ConnectionFlow
+        name={name!}
+        sdk={sdk}
+        walletService={walletService}
+        onCancel={() => setConnectModal('selector')}
+        onSuccess={() => setConnectModal('connectionSucceed')}
+        className="jarvis-styles"
+      />;
+    case 'connectionSucceed':
+      return <ConnectionSuccess hideModal={() => {history.push('/');}}/>;
```
Comment (xitronix) [inline universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx]: Consider histrory.replace?

Comment (JustynaBroniszewska) [reply universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx]: I will

Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/ConnectAccount/ConnectAccount.tsx]:
```suggestion
      return <ConnectionSuccess hideModal={() => {history.replace('/');}}/>;
```

**Comment by JustynaBroniszewska** [comment]
It is not a bug. It is a feature. A user can see once again this wonderful screen.

File: universal-login-wallet/src/ui/react/Modals/ModalTxnSuccess.tsx [hunk 3]
```diff
@@ -28,10 +31,28 @@ export const ModalTxnSuccess = ({hideModal}: ModalTxnSuccessProps) => {
...
 };
+
+interface CreationSuccessProps {
+  hideModal: () => void;
+}
+
+export const CreationSuccess = ({hideModal}: CreationSuccessProps) =>
+  <ModalTxnSuccess
+    hideModal={hideModal}
+    operation='Wallet creation'
```
Comment (Marik-D) [inline universal-login-wallet/src/ui/react/Modals/ModalTxnSuccess.tsx]: Please use "

**Pull request merged**